### PR TITLE
feat: add class-names for push-analytics

### PR DESCRIPTION
**Pull request by HendrikThePendric**
Implements: No issue number available

**Requires https://github.com/dhis2/analytics/pull/1620**

---

### Key features

1. Adds class-names to elements that push-analytics need to interact with

---

### TODO

-   [x] Wait for https://github.com/dhis2/analytics/pull/1620 to be merged and bump analytics version

### CODE REVIEW

File: src/components/DownloadMenu/ToolbarDownloadDropdown.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ const ToolbarDownloadDropdown = () => {
     const { disabled, doDownloadData, doDownloadImage, visType } = useDownload()
 
     return (
-        <HoverMenuDropdown label={i18n.t('Download')} disabled={disabled}>
+        <HoverMenuDropdown
+            label={i18n.t('Download')}
+            disabled={disabled}
+            className="download-dropdownmenu-button"
```
Comment (HendrikThePendric) [inline src/components/DownloadMenu/ToolbarDownloadDropdown.js:14]:
```suggestion
            className="push-analytics-download-dropdown-menu-button"
```

File: src/components/DownloadMenu/TableMenu.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ export const TableMenu = ({ hoverable, onDownload }) => {
             <MenuItemComponent
                 key="html"
                 label={i18n.t('HTML (.html)')}
+                className="download-menuitem"
```
Comment (HendrikThePendric) [inline src/components/DownloadMenu/TableMenu.js:47]:
```suggestion
                className="push-analytics-download-menu-item"
```

File: src/components/DownloadMenu/GraphicsMenu.js [hunk 3]
```diff
@@ -27,6 +27,7 @@ export const GraphicsMenu = ({ hoverable, onDownload }) => {
                 icon={<IconImage24 color={colors.grey600} />}
                 label={i18n.t('Image (.png)')}
                 onClick={() => onDownload({ format: FILE_FORMAT_PNG })}
+                className="download-menuitemn"
```
Comment (HendrikThePendric) [inline src/components/DownloadMenu/GraphicsMenu.js:30]:
```suggestion
                className="push-analytics-download-menu-item"
```

**Comment by HendrikThePendric** [comment]
Note that class name `push-analytics-download-menu-item` is used in both the chart download menu and the table download menu. This is intentional because it allows the download to be triggered in a single way.

**Review by edoardo — APPROVED** [review]

**Pull request merged**